benchmark/python/gluon/benchmark_gluon_rnn.py

```diff
@@ -0,0 +1,225 @@
+        if hybridize:
+            rnn_layer.hybridize()
+        out = rnn_layer(input_data)
+
+    if is_train: 
+        #out = mx.sym.slice(out, begin=(0, None), end=(bs, None))
+        hidden = mx.sym.Reshape(data = out, shape=(-1, hidden_size))
+        pred = mx.sym.FullyConnected(data=hidden, num_hidden=embedding_size, name='pred')
+        if layout == 'TNC':
+            pred = mx.sym.Reshape(data=pred, shape=(seq_len, -1, embedding_size))
+        elif layout == 'NTC':
+            pred = mx.sym.Reshape(data=pred, shape=(-1, seq_len, embedding_size))
+        softmax_output = mx.sym.SoftmaxOutput(data=pred, name='softmax')
+
+    if layout == 'NTC':
+        dshape = (bs, seq_len, embedding_size)
+    elif layout == 'TNC':
+        dshape = (seq_len, bs, embedding_size)
+    
+    if is_train:
+        mod = mx.mod.Module(softmax_output, label_names=('softmax_label',), context=ctx)
+    else:
+        mod = mx.mod.Module(out, label_names=None, context=ctx)
+    
+    if is_train:
+        if layout == 'TNC':
+            mod.bind(for_training = True, data_shapes=[('data', dshape)],
+                label_shapes=[('softmax_label', (seq_len, bs, embedding_size))])
+        elif layout == 'NTC':
+            mod.bind(for_training = True, data_shapes=[('data', dshape)],
+                label_shapes=[('softmax_label', (bs, seq_len, embedding_size))])
+        
+    else:
+        mod.bind(data_shapes=[('data', dshape)], label_shapes=None)
+
+    batch = mx.io.DataBatch(data=[mx.random.uniform(shape=dshape)], label=[])
+    mod.init_params(initializer=mx.init.Xavier(magnitude=2.))
+    if is_train:
+        mod.init_optimizer(optimizer='sgd')
+        mod.forward(batch, is_train=True)
+        if unfuse:
+            for o in mod.get_outputs():
+                o.wait_to_read()
+        else:
+            o = mod.get_outputs()[0]
+            o.wait_to_read()
+        mod.backward()
+        mod.update()
+    else:
+        mod.forward(batch, is_train=False)
+        if unfuse:
+            for o in mod.get_outputs():
+                o.wait_to_read()
+        else:
+            o = mod.get_outputs()[0]
+            o.wait_to_read()
+
+if __name__ == '__main__':
+
+    num_layer = opt.num_layer
+    layout = opt.layout
+    latency = opt.latency
+    hybridize = not(opt.no_hybridize)
+    
+    if layout not in input_layout:
+        logging.warning('Only TNC or NTC are supported!')
+        sys.exit(0)
+
+    if celltype not in rnncell_type:
+        logging.warning('Only LSTM and GRU cell are supported!')
+        sys.exit(0)
+    
+    if celltype == 'all':
+        cell_lst = ['lstm', 'gru']
+    else:
+        cell_lst = [celltype]
+
+    if opt.specify_shape != None:
+        input_shape_list = [[int(x) for x in opt.specify_shape.split(',')]]
+
+    for cell in cell_lst:
+        if opt.train:
+            logging.info('%s training benchmark.', cell)
+        else:
+            logging.info('%s inference benchmark.', cell)
```

Comment:
can this be changed to check for opt.train before the for loop?
something like
`print_string = 'training' if opt.train else 'inference'`
and inside the for loop it would be `logging.info('%s %s benchmark.', cell, print_string)`